### ISSUE TITLE
E701: false positive since 0.0.250

### ISSUE THREAD

**Issue by bastimeyer** [charliermarsh/ruff#3108]
**E701.py**
```py
from typing import Match, Optional


class Foo:
    match: Optional[Match] = None
```

```
$ ruff --version
ruff 0.0.250

$ ruff --select E E701.py 
E701.py:5:10: E701 Multiple statements on one line (colon)
Found 1 error.
```

No issues on `0.0.249`.

This looks like it's related to the `match` class attribute name. If this gets changed to something else, then no error gets raised.

**Comment by charliermarsh** [comment]
Ohh yes. This makes sense. Thank you and sorry for the false positive.

**Comment by charliermarsh** [comment]
Would be solved by https://github.com/RustPython/RustPython/pull/4542 plus a minor change in here.
